packages/block-library/src/media-text/edit.js

```diff
@@ -117,12 +117,13 @@ class MediaTextEdit extends Component {
 		const classNames = classnames( className, {
 			'has-media-on-the-right': 'right' === mediaPosition,
 			'is-selected': isSelected,
+			'has-background': ( backgroundColor.class || backgroundColor.color ),
 			[ backgroundColor.class ]: backgroundColor.class,
 		} );
 		const widthString = `${ temporaryMediaWidth || mediaWidth }%`;
 		const style = {
 			gridTemplateColumns: 'right' === mediaPosition ? `auto ${ widthString }` : `${ widthString } auto`,
-			backgroundColor: backgroundColor.color,
+			backgroundColor: backgroundColor.class ? undefined : backgroundColor.color,
```

Comment:
It was decided to use inline color styles in the editor so even if the theme developer is not loading the editor styles with the color classes, the color is still viewable in the editor. So I will revert this line change during the update.